src/engine/naive_engine.cc

```diff
@@ -19,12 +21,23 @@ class NaiveEngine final : public Engine {
     std::vector<VarHandle> mutable_vars;
     FnProperty prop;
     const char* opr_name;
+    /*! \brief indicate whether to profile this operator */
+    bool profiling{false};
```

Comment:
this is accordant with https://github.com/dmlc/mxnet/blob/nnvm/src/engine/threaded_engine.h#L56, and should add it for both ThreadedEngine and NativeEngine? if need, i'll add both.